Makefile

```diff
@@ -1,12 +1,6 @@
-.PHONY: certs publish test
-
-certs:
-	curl http://ci.kennethreitz.org/job/ca-bundle/lastSuccessfulBuild/artifact/cacerts.pem -o hyper/certs.pem
```

Comment:
What was this command doing? I'm sorta reluctant to remove Make targets just for the sake of it.

Reply:
This line was here since the initial commit in this repo.
No idea what it does - but the URL does not give a valid response and is not usable.